jetty9-base/src/main/assembly/assembly.xml

```diff
@@ -33,12 +33,5 @@
         <exclude>**/META-INF/**</exclude>
       </excludes>
     </fileSet>
-    <fileSet>
-      <directory>${project.build.directory}</directory>
-      <outputDirectory>jetty-base/lib/gae</outputDirectory>
```

Comment:
The resources and dependency plugins are used to obtain and manipulate the inputs to this build. The assembly plugin is used to create the jetty-base tar.gz artefact that is later consumed when creating the jetty9 image.